pkgs/development/tools/godot/export-templates.nix

```diff
@@ -10,6 +10,16 @@ godot.overrideAttrs (oldAttrs: rec {
     mkdir -p "$out/share/godot/templates/${oldAttrs.version}.stable"
     cp bin/godot.x11.opt.64 $out/share/godot/templates/${oldAttrs.version}.stable/linux_x11_64_release
   '';
+  postFixup = ''
+    ${oldAttrs.postFixup or ""}
+
+    # https://docs.godotengine.org/en/stable/development/compiling/optimizing_for_size.html
+    # Stripping reduces the template size from around 500MB to 40MB for Linux.
+    # This also impacts the size of the exported games.
+    # This is done explicitly here because mkDerivation does not automatically
+    # strip binaries in the template directory.
+    strip $out/share/godot/templates/*/*
```

Comment:
I think setting `stripAllList = (oldAttrs.stripAllList or [ ]) ++ [ "share/godot/templates" ]` would produce the same result.